Studio-Lovelies/GG-JointJustice-Unity
# Assimilate all action names to SCREAMING_SNAKE_CASE format

## Summary

Changed the following action names to SCREAMING_SNAKE_CASE so all action names are consistent:
- `SHOWACTOR` -> `SHOW_ACTOR`
- `PLAYSFX` -> `PLAY_SFX`
- `PLAYSONG` -> `PLAY_SONG`
- `SHAKESCREEN` -> `SHAKE_SCREEN`
- `AUTOSKIP` -> `AUTO_SKIP`

NOTE: I will be changing the action name docs once this is merged

## Testing instructions

Test any inky script that uses these actions (or make your own!) and see that the actions still trigger correctly.
These are the current existing inky scripts that use any of these actions:
- `ActorTestInk.ink` - `SHOW_ACTOR` and `PLAY_SFX`
- `ActorTestInk2.ink` - `PLAY_SFX`
- `AppearingDialogInk.ink` - `AUTO_SKIP`
- `AudioTest.ink` - `PLAY_SONG`
- `Ross_Cool_X.ink` - `PLAY_SFX`
- `TestInk.ink` - `SHOW_ACTOR` and `PLAY_SONG`

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
